linters/elixir/.credo.exs

```diff
@@ -0,0 +1,21 @@
+%{
+  configs: [
+    %{
+      name: "default",
+      files: %{
+        included: ["lib/", "src/", "web/", "apps/"],
+        excluded: [~r"/_build/", ~r"/deps/"]
+      },
+      requires: [],
+      check_for_updates: true,
+      strict: true,
+      color: true,
+
+      checks: [
+        {Credo.Check.Design.TagTODO, exit_status: 0},
```

Comment:
I'm actually on the fence on this one and would like a second opinion. `TODO` comments, by default, make `mix credo` exit with status code 2 and the CI fail. Having `exit_status: 0` still makes the check but the CI doesn't fail. Which means when you run `mix credo` it's going to output a warning and remind you. It's not considered a failed check.